tests/cupy_tests/random_tests/test_generator.py

```diff
@@ -1192,6 +1192,19 @@ def test_randint_3(self):
         self.generate(3, 10, size=0)
 
 
+class TestRandintMisc(unittest.TestCase):
+
+    def test_randint_overflow(self):
+        cupy.random.randint(numpy.int8(-100), numpy.int8(100))
+
+    def test_randint_float(self):
+        cupy.random.randint(1.2, 3.4, 5)
```

Comment:
I think we should test this more thoroughly.

* check the range of generated values.
* negative numbers
* `high` is omitted
* conditions whose result would be different if `ceiling` was used